src/main/java/kr/codesqaud/cafe/repository/MemoryArticleRepository.java

```diff
@@ -0,0 +1,32 @@
+package kr.codesqaud.cafe.repository;
+
+import kr.codesqaud.cafe.domain.Article;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Repository
+public class MemoryArticleRepository implements ArticleRepository {
+    private final Map<Long, Article> articleMap = new HashMap<>();
+    private static Long sequence = 0L;
+
+
+    @Override
+    public void save(Article article) {
+        article.setArticleIndex(++sequence);
+        articleMap.put(article.getArticleIndex(), article);
```

Comment:
```suggestion
        articleMap.put(++sequence, article);
```
위와 같이하는 게 더 좋을 것 같습니다.

Reply:
아하 감사합니다 !!
그렇다면 Article 클래스에서 Long형 articleIndex라는 필드를 갖고 있지 않는 게 더 좋을 것 같다는 말씀이실까요??

Reply:
코드 한줄을 삭제할 수 있어요 그렇게 작성해봤어요. 여기에서 index 를 삭제해라는 의미는 없었습니다

Reply:
엇 그러면 article에 있는 articleIndex를 Set하고 Get 과정이 없어지게 된다는 것인데
이렇게 되면 articleIndex가 있는 의미가 없어져버리거든요!
그 의미에서 말씀드렸습니다 😲

Reply:
아하 이해했습니다. 제가 착각했어요.

추가 의견이 있는데 저게 Article 에 대한 식별값이면 Index 라는 네이밍보다 Id 가 더 적합하지 않을까요?